edb/server/main.py

```diff
@@ -810,12 +781,22 @@ def add_config(name, value, type_):
                 value = value.to_backend_str()
             backend_settings[setting.backend_setting] = str(value)
 
+    def iter_environ():
+        translate_env = {
+            "EDGEDB_SERVER_BIND_ADDRESS": "listen_addresses",
+            "EDGEDB_SERVER_PORT": "listen_port"
+        }
+        for name, value in os.environ.items():
+            if cfg := translate_env.get(name):
+                yield name, value, cfg
+            else:
+                cfg = name.removeprefix("EDGEDB_SERVER_CONFIG_cfg::Config.")
```

Comment:
We are planning to deprecate `Config` objects and replace with top-level variables instead, so the variables should be `EDGEB_SERVER_CONFIG_cfg::apply_access_policies`.  IOW, remove `Config.`